### ISSUE TITLE
CLOUDCMD_PREFIX env variable has no effect

### ISSUE THREAD

**Issue by barkap** [coderaiser/cloudcmd#328]
```
$ docker run --rm -p 8000:8000 -e CLOUDCMD_PREFIX=/pfx coderaiser/cloudcmd
url: http://localhost:8000/

$ curl localhost:8000/pfx
ENOENT: no such file or directory, lstat '/usr/src/app/pfx'
```

* **Version**: v15.6.0 (docker)

**Comment by coderaiser** [comment]
Thanks a lot :), fixed with 07fe299. Landed in [v15.7.1](https://github.com/coderaiser/cloudcmd/releases/tag/v15.7.1) 🎉 .
Is it works for you?

**Comment by barkap** [comment]
Yes, it does work. Thanks for a quick fix! ;)
